Classes/PostProcessor/PublishStatusPostProcessor.php

```diff
@@ -32,8 +32,20 @@ public function process($response): array
     {
         $response = (array)$response;
         foreach ($response as $key => $item) {
+            // Skip _root entry
+            if ($key === '_root') {
+                continue;
+            }
+
+            // Check if leistung is a leistung or a zustaendigkeit with a leistung as array item
+            if (array_key_exists('leistung', $item) && is_array($item['leistung'])) {
+                $leistung = $item['leistung'];
+            } else {
+                $leistung = $item;
```

Comment:
Heißt aber, wenn `leistung` als Key existiert aber z.B. ein String ist, dann wandert diese Property nicht in $leistung. Korrekt so?

Reply:
Das ist korrekt so. Normalerweise ist $item ein Array mit Infos zur Leistung, jedoch bei einer Zuständigkeit ist $item eine Zuständigkeit und $item['leistung'] ist das Array mit Infos zur Leistung.